### PR TITLE
Worklets tabbar

### PR DESCRIPTION
**Pull request by karol-bisztyga**

### CODE REVIEW

File: Example/node_modules/react-native/ReactAndroid/build.gradle [hunk 1]
```diff
@@ -256,18 +256,17 @@ def getNdkBuildFullPath() {
 task buildReactNdkLib(dependsOn: [prepareJSC, prepareHermes, prepareBoost, prepareDoubleConversion, prepareFolly, prepareGlog], type: Exec) {
     inputs.dir("$projectDir/../ReactCommon")
     inputs.dir("src/main/jni")
+    inputs.dir("src/main/java/com/facebook/react/modules/blob")
     outputs.dir("$buildDir/react-ndk/all")
-    outputs.upToDateWhen { false }
     commandLine(getNdkBuildFullPath(),
-            "NDK_DEBUG=0",
+            "NDK_DEBUG=" + (nativeBuildType.equalsIgnoreCase("debug") ? "1" : "0"),
```
Comment (Szymon20000) [inline Example/node_modules/react-native/ReactAndroid/build.gradle:262]: Revert changes in node_modules.

File: Example/testComponents/TabBar.js [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+      }, [{ width, tabWidth, tabHeight }, { commands }]
+    );
+    mapper();
+
+    const spring = useSpring({},{});
+
+    const opWorklet = useEventWorklet(function(input) {
+      'worklet'
+      if (this.event.state !== 2) {
+          return true
```
Comment (Szymon20000) [inline Example/testComponents/TabBar.js:139]: Do you really want to unregister this handler?

File: Example/node_modules/react-native/ReactAndroid/build.gradle (hunk 1 above)
Comment (karol-bisztyga) [reply Example/node_modules/react-native/ReactAndroid/build.gradle]: fixed 3382a43

File: Example/testComponents/TabBar.js (hunk 2 above)
Comment (karol-bisztyga) [reply Example/testComponents/TabBar.js]: no, fixed f63aa4b

**Pull request merged**